### DISCUSSION TITLE
upload file txt on page and it redirect to another page

### DISCUSSION THREAD

**Discussion by tampro9x92** [requarks/wiki#3311] [Error / Bug Report]
Go to create page 
Chose Visual Editor
Chose insert Assets in the menu bar of editor
Chose Browse... in the Upload Assets then chose a file have extension is .txt then chose upload in table assets chose the file have extension .txt then click insert then click on Create btn then close go to the go to the page that you just create click on the link file you just insert it will blow your mind

**Comment by rb-oscu** [comment]
I have the same issue. If you attach a txt file using the visual editor (insert assets) and then try to click the attachment link on the page, it will ask you to create a new page instead of opening or downloading the .txt file.

**Comment by biganan2012** [comment]
I have faced the same problem. I am using docker-compose and markdown type.

**Reply by biganan2012** [reply]
Extensions .md, .html and .txt are reserved for pages. Either zip or change the extension of files to get them as downloads.

**Comment by sria53** [comment]
If you do not need to use txt for pages go Administration-General-URL Handling and remove txt from page extensions.
